### PR TITLE
Document Token Auth Method for Beyond Identity

### PR DESCRIPTION
**Pull request by turnipdabeets**

### CODE REVIEW

File: docs/using-bi-for-auth.md [hunk 1]
```diff
@@ -75,16 +75,39 @@ Use the URL above from a browser, and watch how the authentication process unfol
...
+![Screenshot](./screenshots/TokenUrl.jpg)
 
-The following request parameters can be POSTed as a part of the URL-encoded form values to the API:
+You'll also need to make note of your application's Token Endpoint Auth Method.
 
-```bash
-grant_type=authorization_code&
+![Screenshot](./screenshots/TokenAuthMethod.png)
+
+Client Secret Basic:
```
Comment (ismailmustafa) [inline docs/using-bi-for-auth.md:86]: I would make these a header, probably 4 (####).

File: docs/using-bi-for-auth.md [hunk 2]
```diff
@@ -75,16 +75,39 @@ Use the URL above from a browser, and watch how the authentication process unfol
...
-&redirect_uri=<redirect_uri_must_match_value_used_in_authorization_request>
-```
\ No newline at end of file
+&redirect_uri=<redirect_uri_must_match_value_used_in_authorization_request>"
+```
+
+Client Secret Post:  
+The client_id and client_secret are sent in the body of the POST request as a form parameter.
+
+```
```
Comment (ismailmustafa) [inline docs/using-bi-for-auth.md:102]: Let's add `bash` syntax highlighting to these.

File: docs/using-bi-for-auth.md (hunk 1 above)
Comment (turnipdabeets) [reply docs/using-bi-for-auth.md]: updated!

File: docs/using-bi-for-auth.md (hunk 2 above)
Comment (turnipdabeets) [reply docs/using-bi-for-auth.md]: updated!

**Review by ismailmustafa — APPROVED** [review]

**Pull request merged**